test/ModalSpec.js

```diff
@@ -27,21 +40,30 @@ describe('Modal', function () {
       render() {
         return (
           <div>
-            <Modal show={this.state.modalOpen} onHide={this.handleCloseModal} container={this}>
+            <Modal
+              animation={false}
+              show={this.state.modalOpen}
+              onHide={this.handleCloseModal}
+              container={this}
+            >
               <strong>Message</strong>
             </Modal>
           </div>
         );
       }
     });
-    let instance = ReactTestUtils.renderIntoDocument(
-        <Container />
-    );
+    let instance = render(
+          <Container />
+        , mountPoint);
+
+    let modal = ReactTestUtils.findRenderedComponentWithType(instance, Modal);
+
     assert.ok(React.findDOMNode(instance).className.match(/\modal-open\b/));
```

Comment:
the old one, unnoticed semi-bug :smile: 
`/\modal-open\b/` => `/\bmodal-open\b/`